src/python/pants_release/generate_github_workflows.py

```diff
@@ -1206,6 +1206,17 @@ def release_jobs_and_inputs() -> tuple[Jobs, dict[str, Any]]:
                         """
                     ),
                 },
+                {
+                    "name": "Trigger cheeseshop build",
+                    "env": {
+                        "GH_TOKEN": "${{ secrets.WORKER_PANTS_CHEESESHOP_TRIGGER_PAT }}",
+                    },
+                    "run": dedent(
+                        """\
+                        gh api -X POST "/repos/pantsbuild/wheels.pantsbuild.org/dispatches" -F event_type=github-pages
```

Comment:
A cherry-on-top improvement (not required) might be to pass through some metadata here, so that we can match up builds between the two repos more easily, e.g. have an input that is set to the version number (or something) that can be used to set https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#run-name

Definitely not required, though.

Reply:
The other repo always builds from scratch, so knowing which version triggered a rebuild, albeit interesting, might not really be useful

Reply:
Ah yeah, I guess we can just check timestamps to confirm the builds ran at all.

Reply:
But actually, I like you link. I'll try and see if we can get that working.

Reply:
I'm having trouble coming up with a nice way to wire it up together. I'll put this on my backlog and we can circle back around.